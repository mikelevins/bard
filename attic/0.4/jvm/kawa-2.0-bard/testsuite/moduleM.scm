;; A runnable module with module-extends and a body that prints something.
(module-extends SimpleB)
(define xx ::integer 20)
(define (get-f) (invoke (this) 'f 5))
(format #t "Hello world - ~d ~a!~%" xx (get-f))
