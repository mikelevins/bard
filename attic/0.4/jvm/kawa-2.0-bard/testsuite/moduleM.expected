Hello world - 20 1035!
